atd-vze/src/Components/GridTableSearch.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState, useEffect } from "react";
+import { withApollo } from "react-apollo";
+import {
+  Col,
+  Button,
+  Form,
+  FormGroup,
+  Input,
+  InputGroup,
+  InputGroupAddon,
+  InputGroupButtonDropdown,
+  DropdownToggle,
+  DropdownItem,
+  DropdownMenu,
+  Alert,
+} from "reactstrap";
+
+// TODO add query operators to each field to better fit data types (_eq, etc.)?
+const GridTableSearch = ({ query, clearFilters, setSearchParameters }) => {
+  const [searchFieldValue, setSearchFieldValue] = useState("");
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [fieldToSearch, setFieldToSearch] = useState("");
+  const [isFieldSelected, setIsFieldSelected] = useState(false);
+
+  const fieldsToSearch = query.searchableFields;
+
+  /**
+   * Handles the submission of our search form
+   * @param {object} e - the event object
+   */
+  const handleSearchSubmission = e => {
+    e.preventDefault();
+
+    setSearchParameters({
+      column: fieldToSearch,
+      value: searchFieldValue,
+    });
+  };
+
```

Comment:
I found a bug related to search + pagination. If click through and paginate a few pages, then do a search with limited results, you end up in a state where you're on page 3 out 1. And you have to paginate page two pages to see your search results. Its it possible to reset the pagination after a new search?

<img width="1552" alt="Screen Shot 2019-09-12 at 12 14 07 AM" src="https://user-images.githubusercontent.com/5697474/64755956-93e5dd00-d4f2-11e9-91a6-2a82455b586f.png">